### PR TITLE
Our projects page completed

### PR DESCRIPTION
**Pull request by jayp822**
Created about/projects directory page. Didn't have title, description, image, and link so I just used temp data. Made a more general carousel component that can be used by others in other parts of the website as well. 

https://github.com/user-attachments/assets/db672f3f-6d61-406d-899e-cd0ae82bfbc1

### CODE REVIEW

**Review by justinquinnb — COMMENTED** [review]
Temp data is fine and the page looks good. We'll get real data in there as we begin the project this year. For the Carousel Component, can we add some spacing between carousel items and adjust their sizing to match the Figma? The card at the center should be bigger than the next and previous items in the carousel, visible on the left and right. When the carousel changes to the next card, they should slide but also grow/shrink appropriately. If possible, the carousel component should accept 1 to an arbitrary number of items, not sure if this functionality is already implemented.

Otherwise great work yet again, grateful for all your help in such a short time frame!

**Comment by jayp822** [comment]
@justinquinnb yeah I can work on it just realized there is no background 😅.

**Comment by justinquinnb** [comment]
Nw! Really appreciate your contributions, your help hasn't gone unnoticed 😀

**Comment by nivedhanatarajan7** [comment]
The Projects page looks great! Since we will begin working on the first DevDogs project (Optimized Schedule Builder) this year, I think we can comment the section related to past projects and just display the current project section.

**Comment by jayp822** [comment]
Removed Stock images used in old home carousel and updated our projects a carousel to look more like the one in figma. Currently the past projects carousel works but is commented out noted by @nivedhanatarajan7 .

https://github.com/user-attachments/assets/bd154845-8a46-4884-a65d-c42fefa33762

File: src/app/about/projects/page.js [hunk 1]
```diff
@@ -1,9 +1,114 @@
...
+    <h2 className=" font-bold text-[2.5rem] sm:text-[3.5rem] md:[4rem] text-UGA inline">
+     Current{" "}
+    </h2>
+    <h2 className="font-bold text-[2.5rem] sm:text-[3.5rem] md:[4rem] text-black inline">
+     Project
+    </h2>
+   </div>
+   {/* Carousel */}
+   {/* <CarouselAdvanced /> */}
+   <EmblaCarousel slides={currentProjects} options={OPTIONS} />
```
Comment (nivedhanatarajan7) [inline src/app/about/projects/page.js:78]: I think we will probably be working on one project as a group at DevDogs each year or semester, so I dont think the display of the current project needs to be a carousel. It can be a picture card.

File: src/app/about/projects/page.js [hunk 2]
```diff
@@ -1,9 +1,114 @@
...
+
+
+   <p className="pt-10 text-base sm:text-lg md:text-xl">
+    DevDogs is proud to consist of ## UGA students from all tech disciplines
```
Comment (nivedhanatarajan7) [inline src/app/about/projects/page.js:82]: This section would contain the description of the project, so we can keep the text here as a placeholder while we get the project description ready. The number of students in DevDogs is something that can change, so I think we can just say "...consist of UGA student..." Stating "from all tech disciplines" seems like a strong statement, so I think we can reword this to be "...from many tech disciplines..." Thank you!

File: src/app/about/projects/page.js [hunk 3]
```diff
@@ -1,9 +1,114 @@
...
+    and backgrounds. We firmly believe that different perspectives breed
+    the best solutions--and we think our work shows! Take a look at some of
+    the awesome people that have made DevDogs, DevDogs below.
+   </p>
+   <Link
+    target="_blank"
+    href="https://github.com/DevDogs-UGA"
```
Comment (nivedhanatarajan7) [inline src/app/about/projects/page.js:89]: The View button should probably link to the Our People page since the paragraph above is about the members of the team.

File: public/Screenshot 2024-08-17 at 4.41.35 PM.png
Comment (nivedhanatarajan7) [inline public/Screenshot 2024-08-17 at 4.41.35 PM.png:1]: Please rename this file with a meaningful name.

**Review by nivedhanatarajan7 — CHANGES_REQUESTED** [review]
The Project page looks good. I gave some comments regarding styling and content. It would also be nice if you could add a comment in the various carousel components since there are different components with probably varying purposes.

File: src/app/about/projects/page.js (hunk 2 above)
Comment (jayp822) [reply src/app/about/projects/page.js:82]: I would be pretty cool if we had a live counter from our discord that could send the data to the webpage and update it in the background whenever someone renew joins. Could be a cool addition for the future

File: src/app/about/projects/page.js (hunk 1 above)
Comment (jayp822) [reply src/app/about/projects/page.js:78]: Ok, yeah I also agree that the projects page should be like a showcase for our main project for the semester so more people are on the same page about the main goal. I could then just have like learn more button or make the card clickable so it can redirect to the github repository if you want.

Comment (nivedhanatarajan7) [reply src/app/about/projects/page.js:78]: We plan to have a page about the Schedule Builder project, and I will probably make that once the details of the project have been confirmed. For now, there is a View button in the Projects page, that you can link to the GitHub repository for the project.

**Comment by jayp822** [comment]
@nivedhanatarajan7 should this button redirect to "/about/projects" or stay as the discord link?
suggestion

**Comment by nivedhanatarajan7** [comment]
The All Projects button should link to the Projects page. Thank you!

**Comment by jayp822** [comment]
Completed edits with this new addition
https://github.com/user-attachments/assets/a64cc531-bb1f-49c5-b4ab-a78d8e33d8cc

**Review by nivedhanatarajan7 — APPROVED** [review]
